compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -591,6 +591,73 @@ tests tools@Tools{damlc} = testGroup "Packaging" $
             ]
           buildProjectError projC "" "dependencies with same unit id but conflicting package ids: a-0.0.1"
 
+    , testCaseSteps "Error on newer LF data-dependency" $ \step -> withTempDir $ \tmpDir -> do
```

Comment:
These tests will break once we make DAML-LF 1.8 the default target, won't they? Shall we nail down the second target to DAML-LF 1.7 to prevent this?

Reply:
oops, I thought I had done that but apparently I didn’t. I’ll fix it. Thanks!